third_party/proto/ibc/applications/transfer/v2/token.proto

```
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package ibc.applications.transfer.v2;
+
+option go_package = "github.com/cosmos/ibc-go/v9/modules/apps/transfer/types";
+
+import "ibc/applications/transfer/v1/transfer.proto";
```

Buf (1.47.2)

import "ibc/applications/transfer/v1/transfer.proto": file does not exist

(COMPILE)